providers/cncf/kubernetes/tests/unit/cncf/kubernetes/hooks/test_kubernetes.py

```diff
@@ -760,9 +760,7 @@ def test_check_kueue_deployment_running(self, mock_get_deployment_status, mock_l
     @mock.patch(HOOK_MODULE + ".KubernetesHook.log")
     @mock.patch(HOOK_MODULE + ".KubernetesHook.get_deployment_status")
     def test_check_kueue_deployment_raise_exception(self, mock_get_deployment_status, mock_log):
-        mock_get_deployment_status.side_effect = ValueError(
-            "Exception occurred while checking for Deployment status."
-        )
+        mock_get_deployment_status.side_effect = ValueError
 
         with pytest.raises(ValueError, match="Exception occurred while checking for Deployment status."):
```

Comment:
There is no period in the code that triggers this, its a colon.   Fell for the trap that @uranusjr pointed out in the other PR